### ISSUE TITLE
Fail to get Montezuma results in the paper

### ISSUE THREAD

**Issue by zhanghc12** [oxwhirl/opiq#1]
I trained Montezuma according to the command in README, but got 0 points after 8.5million samples:

INFO - OptimisticAction - Epsilon: 0.01
WARNING - Main - T: 8,537,435, Episode Reward: 0.00
WARNING - Main - T: 8,537,681, Episode Reward: 0.00

I did not change hyperparameters.

**Comment by tabzraz** [comment]
Sorry about this, I checked the hyperparameters that were used for the experiments and count_beta was incorrectly set to 0.01 instead of 0.1. I've updated the README with the correct value.
